apps/admin_api/test/admin_api/v1/controllers/token_controller_test.exs

```diff
@@ -76,6 +77,38 @@ defmodule AdminAPI.V1.TokenControllerTest do
     end
   end
 
+  describe "/token.stats" do
+    test "returns the stats for a token" do
+      token = insert(:token)
+      _mints = insert_list(3, :mint, token_uuid: token.uuid)
```

Comment:
Can we add `%{amount: 100_000}` to `insert_list()` so it's clear that we're asserting 300_000 because we're minting 100_000 three times? 

Also prevents test failing if the factory amount ever changes.